packages/did/src/Did.chain.ts

```diff
@@ -60,36 +62,27 @@ export type ChainDidPublicKeyDetails = DidDidDetailsDidPublicKeyDetails
 
 // ### RAW QUERYING (lowest layer)
 
-export function encodeDid(did: DidUri): KiltAddress {
+export function didToChain(did: DidUri): KiltAddress {
   return parseDidUri(did).address
 }
 
-// Query a full DID.
-// Interacts with the DID storage map.
-async function queryDidEncoded(did: DidUri): Promise<Option<DidDidDetails>> {
-  const api = ConfigService.get('api')
-  return api.query.did.did(encodeDid(did))
+export function resourceIdToChain(id: UriFragment): string {
+  return stripFragment(id)
 }
 
-// Query ALL deleted DIDs, which can be very time-consuming if the number of deleted DIDs gets large.
-async function queryDeletedDidsEncoded(): Promise<GenericAccountId[]> {
-  const api = ConfigService.get('api')
-  // Query all the storage keys, and then only take the relevant property, i.e., the encoded DID address.
-  const entries = await api.query.did.didBlacklist.keys()
-  return entries.map(({ args: [encodedAddresses] }) => encodedAddresses)
+export function deletedDidsFromChain(
+  encoded: Array<StorageKey<[AccountId32]>>
+): DidUri[] {
+  return encoded.map(({ args: [address] }) =>
+    getFullDidUri(address.toHuman() as KiltAddress)
+  )
 }
```

Comment:
a) I wouldn't use toHuman
b) I don't think we should even have that function, as querying all the deleted Dids is a super fringe edgecase

Reply:
yes, as long as the resolver is there, I don't think we need it

Reply:
Removed in https://github.com/KILTprotocol/sdk-js/pull/613/commits/b9679e1d87997136664246f2fe7b60557d4c15aa